src/workflows/airqo_etl_utils/bigquery_api.py

```diff
@@ -69,6 +69,15 @@ def get_devices_hourly_data(
         self,
         day: datetime,
```

Comment:
Hi @NicholasTurner23 , thanks for this.
This file should ideally not be changed in this PR for analytics microservice. Please crosscheck your Git.